javascript/javascript1/week3/smartphone-usages.js

```diff
@@ -0,0 +1,37 @@
+const activities = [];
+const aDate = new Date();
+const getTheDate = aDate.getDate();
+const getTheMonth = aDate.getMonth() + 1;
+const getTheYear = aDate.getFullYear();
+const today = `${getTheDate}/${getTheMonth}/${getTheYear}`;
+const limitUsage = 60;
+const totalLength = activities;
+
+function addActivity(date, activity, duration) {
+  activities.push({ date, activity, duration });
+}
+
+addActivity("1/11/2022", "this is date 1", 5);
+addActivity(today, "check email", 20);
+
+
+function showStatus(activities) {
+  let sum = 0;
+  for (element of activities) {
+    sum += element.duration;
+  }
+
+  if (showStatus && totalLength.length > 0) {
```

Comment:
Why are you checking the value of `showStatus` inside `function showStatus()`. Also it is a side effect function, it does not return anything. Also why totalLength.length is same as activities.length. no need to define a new variable, just use activities.